src/components/SMARTBox/PatientBox.js

```diff
@@ -316,11 +318,30 @@ export default class PatientBox extends Component {
       })
       .then(result => {
         this.setState({ questionnaireResponses: result });
+      })
+      .then(() => this.getQuestionnaireTitles());
+  }
+
+  getQuestionnaireTitles() {
+    const client = FHIR.client(this.props.params);
+    const promises = [];
+    if (this.state.questionnaireResponses.data.length > 0) {
+      for (const canonical of this.state.questionnaireResponses.data.map(
+        questionnaireResponse => questionnaireResponse.questionnaire
+      )) {
+        promises.push(
+          client.request(canonical).then(questionnaire => [canonical, questionnaire.title])
```

Comment:
Questionnaire title is an optional field I believe.  This won't fail it would just return undefined, but maybe we want to default to the url, or id, or just "No title" in that case.

Reply:
I agree with @KeeyanGhoreshi, we should have a default title if no title is provided. I like the idea of using the url/id

Reply:
@KeeyanGhoreshi @smalho01 did you mean QuestionnaireResponse.questionnaire for the url or Questionnaire.url?